### DISCUSSION TITLE
autoref不支持对章节进行交叉引用

### DISCUSSION THREAD

**Discussion by TsingWei** [sikouhjw/gdutthesis#124] [General]
我想告诉读者去看某一节，所以当我把label放在某一章或节标题下并使用autoref去引用时，得到的还是section x.x，比如图里这样：
![Screenshot_2024-04-30-18-20-40-026_com.microsoft.emmx.jpg](https://github.com/sikouhjw/gdutthesis/assets/28723968/d55683c9-136e-4fc6-bcaf-755508b753c3)

我也尝试自己在autoref规则表中加入对章或节的规则，但是只会得到 小节x.x，而不是中文语境下的x.x小节。请问作者有计划支持吗？

**Comment by sikouhjw** [comment]
不好弄，目前需要手动写 `第~\ref{sec:xxx} 节`
